local_sales/Gemfile.lock

```diff
@@ -62,6 +62,7 @@ GEM
       zeitwerk (~> 2.3)
     addressable (2.8.0)
       public_suffix (>= 2.0.2, < 5.0)
+    bcrypt (3.1.16)
```

Comment:
I recommend you to be more specific in your commit messages and descriptions

Why are you adding those dependencies?
What are you gonna do with those dependencies?

Reply:
And since you are not doing changes in your Gemfile, this is an error in your last PR because you didn't run `bundle install`